src/panels/config/integrations/integration-panels/zwave/zwave-migration.ts

```diff
@@ -79,259 +95,275 @@ export class ZwaveMigration extends LitElement {
-                                </div>
-                              `
-                            : ``}
-                        </div>
-                        <div class="card-actions">
-                          <mwc-button @click=${this._stopNetwork}>
-                            Stop Network
-                          </mwc-button>
-                        </div>
-                      </ha-card>
-                    `
-                  : ``}
-                ${this._step === 2
-                  ? html`
-                      <ha-card class="content" header="Set up OZWDaemon">
-                        <div class="card-content">
-                          <p>Now it's time to set up the OZW integration.</p>
-                          ${isComponentLoaded(this.hass, "hassio")
-                            ? html`
-                                <p>
-                                  The OZWDaemon runs in a Home Assistant addon
-                                  that will be setup next. Make sure to check
-                                  the checkbox for the addon.
-                                </p>
-                              `
-                            : html`
-                                <p>
-                                  If you're using Home Assistant Core in Docker
-                                  or a Python venv, see the
-                                  <a
-                                    href="https://github.com/OpenZWave/qt-openzwave/blob/master/README.md"
-                                    target="_blank"
-                                    rel="noreferrer"
+          ${html`
+            ${this._step === 0
+              ? html`
+                  <ha-card class="content" header="Introduction">
+                    <div class="card-content">
+                      <p>
+                        This wizard will walk through the following steps to
+                        migrate from the legacy Z-Wave integration to Z-Wave JS.
+                      </p>
+                      <ol>
+                        <li>Stop the Z-Wave network</li>
+                        <li>
+                          <i
+                            >If running Home Assistant Core in Docker or in
+                            Python venv:</i
+                          >
+                          Configure and start Z-Wave JS
+                        </li>
+                        <li>Set up the Z-Wave JS integration</li>
+                        <li>
+                          Migrate entities and devices to the new integration
+                        </li>
+                        <li>Remove legacy Z-Wave integration</li>
+                      </ol>
+                      <p>
+                        <b>
+                          Please take a backup of your environment before
+                          proceeding.
+                        </b>
```

Comment:
Let's link to the supervisor backup panel if the `hassio` integration is loaded.